.github/workflows/scorecards.yml

```
@@ -0,0 +1,54 @@
+name: Scorecards supply-chain security
+on:
+  # Only the default branch is supported.
+  branch_protection_rule:
+  schedule:
+    - cron: '24 8 * * 6'
```

This fails our lint checks. Passing version:

```suggestion
    - cron: "24 8 * * 6"
```